static/sass/patterns/_text-blocks.scss

```diff
@@ -0,0 +1,53 @@
+// Text Blocks
+// ===========
+
+.text-block {
+  @include contentblock;
+}
+
+.textblock-headline {
+  @include contentblock-highlight('normal');
+}
+
+.textblock-content {
+  @include contentblock-copy;
+
+  p {
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+.headline-contrast {
+  margin-top: 0;
+
+  @include above('small-page') {
+    text-align: right;
+  }
+
+  span {
+    display: block;
+  }
+
+  .title-main {
+    font-size: size('headline-highlight-large');
+  }
+
+  .fancy-headline-small {
```

Comment:
Do these selectors need to be nested?

Reply:
some properties do since they are overriding other declared (headline) properties, but some don't. I wasn't sure if I should nest them all for consistency or leave them unnested and nest only the override in each block.

Reply:
oh, is it a specificity issue? I think you made the right call keeping things together, if there's not a simple way to adjust things so there aren't conflicts.

Reply:
not anymore. maybe only with that title-main and now that is gone.